### ISSUE TITLE
How can I train a model with 1024px image resolution results?

### ISSUE THREAD

**Issue by Damarcreative** [geocine/sd-easy-mode#22]
How can I train a model with an image resolution of 1024px?
should I change `--resolution=512 \` to `--resolution=1024 \` and I upload the image with 1024 resolution too??

!accelerate launch --num_processes=1 --num_machines=1 --mixed_precision="no" --dynamo_backend="no" --num_cpu_threads_per_process=1 train_dreambooth.py \
    --pretrained_model_name_or_path=$MODEL_NAME \
    --pretrained_vae_name_or_path="stabilityai/sd-vae-ft-mse" \
    --output_dir=$OUTPUT_DIR \
    --revision="fp16" \
    --with_prior_preservation --prior_loss_weight=1.0 \
    --seed=1337 \
    --resolution=512 \
    --train_batch_size=1 \
    --train_text_encoder \
    --mixed_precision="fp16" \
    --use_8bit_adam \
    --gradient_accumulation_steps=1 \
    --learning_rate=1e-6 \
    --lr_scheduler="constant" \
    --lr_warmup_steps=0 \
    --num_class_images=$NUM_CLASS_IMAGES \
    --sample_batch_size=$SAMPLE_BATCH_SIZE \
    --max_train_steps=$MAX_TRAIN_STEPS \
    --save_interval=$SAVE_INTERVAL \
    --save_min_steps=$SAVE_MIN_STEPS \
    --save_sample_prompt="$SAVE_SAMPLE_PROMPT" \
    --concepts_list="concepts_list.json"
